tic_tac_toe/game.py

```diff
@@ -87,18 +128,54 @@ def move(game, player, position):
     checks before the actual movement is done.
     After registering the movement it must check if the game is over.
     """
-    pass
-
-
+    if not _position_is_valid(position):
+        raise InvalidMovement("Position out of range.")
+
+    if _board_is_full(game['board']) or game['winner'] is not None:
```

Comment:
I think it should be sufficient to test `if _board_is_full(game['board']) or game['winner']:`, since `game['winner']` will only ever be `None` or the winner of the game.